AK/StringUtils.cpp

```diff
@@ -519,7 +519,7 @@ ByteString invert_case(StringView str)
 }
 
 // Finishes the replacing algorithm once it is known that ita least one
-// replacemnet is going to be done. Otherwise the caller may want to follow a
+// replacements is going to be done. Otherwise the caller may want to follow a
```

Comment:
replacement

Reply:
Also, at least

Reply:
Ok. I also fixed that. Worth mentioning that typos tool didn't find 'ita'.